tools/pre_commit.py

```diff
@@ -116,7 +116,9 @@ def generate_workflows(ctx: Context):
             "prepare_workflow_needs": NeedsTracker(),
             "build_repo_needs": NeedsTracker(),
         }
-        shared_context_file = tools.utils.REPO_ROOT / "cicd" / "shared-context.yml"
+        shared_context_file = (
+            tools.utils.REPO_ROOT / "cicd" / "shared-gh-workflows-context.yml"
+        )
```

Comment:
There's another reference to shared-context.yml here: https://github.com/saltstack/salt/blob/ffe9c9d1e0981c7e2e337b150ac16ed68dda6048/tools/pkg/build.py#L31

Reply:
Pushed a fix